pre-docs/writing_plans.md

```diff
@@ -278,6 +278,7 @@ A `ResultSet` has the following methods:
 -   `error_set()`: A `ResultSet`containing only the results of failed nodes.
 
 -   `ok_set()`: A `ResultSet` containing only the successful results.
+-   `filter_set(block)`: Filters a `ResultSet` with the given block and returns a `ResultSet` object (where [Puppet's Iterable](https://puppet.com/docs/puppet/6.4/puppet-6.4/content/documents/functions_iterative.html) `filter` returns an array)
```

Comment:
This could be the wrong docs, but I think technically filter can return any type? https://puppet.com/docs/puppet/6.4/function.html#filter